charts/datadog/templates/_container-agent.yaml

```diff
@@ -93,6 +93,9 @@
       subPath: install_info
       mountPath: /etc/datadog-agent/install_info
       readOnly: true
+    - name: tmpdir
```

Comment:
should we shared the same empty dir across containers?

Reply:
I don't see a specific issue except if 2 agents process generate the same file, like a flare.

Reply:
Indeed, it theory it could cause some issues. though creating one volume (emptyDir) / container may be a bit complex in terms of setup.

Reply:
I think it is ok to make it like you did because when the agent is installed on the host, the agents also shared the same `/tmp` folder